.github/workflows/main.yaml

```diff
@@ -0,0 +1,31 @@
+#
+# GitHub actions Python based test runs.
+#
+# Available VMs
+# https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners
+#
+name: GHA-Python
+
+# Only run on pull requests and only for Python files.
+on:
+  push:
+    branches: [ master ]
```

Comment:
Why specify `master` branch here?  Twisted doesn't use a `master` branch.

There is a fuller example here: https://docs.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#starting-with-the-python-workflow-template